package.json

```diff
@@ -13,79 +13,76 @@
   "author": "Product design and research",
   "repository": {
     "type": "git",
-    "url": "https://github.com/AlaskaAirlines/auro-input"
+    "url": "https://github.com/AlaskaAirlines/ods-inputtext"
   },
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "lit-element": "^2.4.0",
-    "chalk": "^4.1.0"
+    "@alaskaairux/icons": "^3.3.0",
+    "chalk": "^4.1.0",
+    "lit-element": "^2.4.0"
   },
   "peerDependencies": {
     "@alaskaairux/orion-design-tokens": "^2.12.1",
-    "@alaskaairux/orion-web-core-style-sheets": "^2.12.1",
-    "@alaskaairux/icons": "^4.0.1",
+    "@alaskaairux/orion-web-core-style-sheets": "^2.12.0",
```

Comment:
Are all these downgrades intentional? Seems like this file might have taken the ods-inputtext version instead of auro-input